setup.cfg

```diff
@@ -38,8 +38,9 @@ setup_requires =
 install_requires =
     python-hookman
     typing_inspect
-    strictyaml
+    strictyaml<1.4
+    ruamel.yaml<0.16.13
```

Comment:
I did return the ruamel pin, `0.16.13` was causing failures (aparentlly interface changes).
Note that strictyaml in `1.4.3` in now bundling ruamel `0.16.12`.